Formula/passenger.rb

```diff
@@ -26,6 +28,20 @@ def install
 
     system "rake", "apache2"
     system "rake", "nginx"
+    nginx_addon_dir = `./bin/passenger-config about nginx-addon-dir`.strip
+
+    mkdir "nginx" do
+      system "tar", "-xf", "#{Formula["nginx"].opt_share}/src/src.tar.xz", "--strip-components", "1"
```

Comment:
Are you sure `--strip-components` is still required here given how the tarball is created above? No worries f so, just wanted to check.

Reply:
`tar` will try to change the last-modified time on `pwd` if I leave it out; if that's ok, then it can be left out.

Reply:
Have no preference so let's leave as-is 👍